recipes-core/rcu-hostname/files/rcuhostname

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
```

Comment:
Can you put a comment on top of this file to mention the usage/purpose of this file?